Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM fedora:20
+FROM centos:7
 MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 
-# Execute system update
-RUN yum -y update && yum clean all
```

Comment:
I do not necessarily agree with you. It's about shipping newest available packages. There is no reason to not do it. Updates fix a lot of issues. We're not respinning base images just because on or other lib has issue. Getting them using yum is way faster + there is no harm, so we should do it.